README.md

```diff
@@ -4,9 +4,72 @@ The purpose of this project is to use SQL as an exploratory tool, towards this e
 corresponding tables and import datasets from [ourworldindata.org](www.ourworldindata.org). Finally we'll analyse this 
 information using SQL.
 
-This project was developed following the steps detailed below.
+This project is divided in two parts:
 
-## 1 - Identify the data requirements
+1. Creation of the Database and tables. Data population in the tables.
+    * DB_creation_and_loading/creation_database.py
+    * DB_creation_and_loading\populate_tables.py
+2. Queries on this database.
+    * Data_exploration\data_exploration_caloric_supply.ipynb
+    * Data_exploration\data_exploration_food_expenditure.ipynb
+    * Data_exploration\data_exploration_gdp.ipynb
+    * Data_exploration\data_exploration_protein_consumption_part1.ipynb
+    * Data_exploration\data_exploration_protein_consumption_part2.ipynb
```

Comment:
Change the case of the files:
`DB_creation_and_loading` -> `DB_CREATION_AND_LOADING`